components/Form.js

```diff
@@ -28,6 +28,7 @@ export default function CreateForm({ onCreateItem }) {
         id="name"
         type="text"
         placeholder="(Required)"
+        maxLength="20"
```

Comment:
That works great now. I would also suggest a maxLength for the comment. But that doesn't really concern this user story. 🤐